cmd/ci-secret-bootstrap/main.go

```diff
@@ -191,8 +191,8 @@ func (o *options) validateCompletedOptions() error {
 					if data.BWItem == "" {
 						return fmt.Errorf("config[%d].from[%s]: bw_item is missing", i, key)
 					}
-					if data.RegistryURLBitwardenField == "" {
-						return fmt.Errorf("config[%d].from[%s]: registry_url_bw_field is missing", i, key)
+					if data.RegistryURLBitwardenField == "" && data.RegistryURL == "" {
```

Comment:
Nit: Can you also validate that only one is set?

Thanks for making this change!

Reply:
Updated.